legend-engine-shared-core/src/main/java/org/finos/legend/engine/plan/platform/java/JavaSourceHelper.java

```diff
@@ -277,4 +278,20 @@ private static boolean fileHasContent(Path path, byte[] content) throws IOExcept
     {
         return (Files.size(path) == content.length) && Arrays.equals(content, Files.readAllBytes(path));
     }
+
+    public static String serviceElementPathToJavaPath(String packagePrefix, String servicePath)
```

Comment:
This is similar as the logic [SDLC uses to generate the class name](https://github.com/finos/legend-sdlc/blob/19fcd19f51e2ae9baa7bff53152658e740abbf8f/legend-sdlc-generation-service/src/main/java/org/finos/legend/sdlc/generation/service/ServiceExecutionClassGenerator.java#L116)

If we go with this direction, will update SDLC to use this method rather than having duplicated logic.